CorsixTH/Lua/ui.lua

```diff
@@ -803,7 +804,7 @@ function UI:afterLoad(old, new)
 
   if old < 100 then
     self:removeKeyHandler({"alt", "enter"}, self)
-    self:addKeyHandler({"alt", "Return"}, self, self.toggleFullscreen)
+    self:addKeyHandler({"alt", "return"}, self, self.toggleFullscreen)
```

Comment:
I'm not sure why you changed this line, or the other place you substituted Return for return. It's not documented in your commit.